### ISSUE TITLE
Type error in Shard.ts

### ISSUE THREAD

**Issue by schuelermine** [ihack2712/shard#3]
```
$ deno --unstable
Deno 1.12.2
exit using ctrl+d or close()
> const Shard = await import("https://deno.land/x/shard@2.0.6/mod.ts");
Check https://deno.land/x/shard@2.0.6/mod.ts
Uncaught TypeError: TS2345 [ERROR]: Argument of type 'number | Date' is not assignable to parameter of type 'string | number | bigint | boolean'.
  Type 'Date' is not assignable to type 'string | number | bigint | boolean'.
    Type 'Date' is not assignable to type 'bigint'.
				._timestamp(BigInt(options?.timestamp ?? Date.now()));
				                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    at https://deno.land/x/shard@2.0.6/lib/Shard.ts:119:24
    at async <anonymous>:2:15
```

**Comment by ihack2712** [comment]
Will get to this after work, thanks!

**Comment by schuelermine** [comment]
I made a PR that fixes this #4
